packages/calcite-components-react/README.md

```diff
@@ -38,6 +40,8 @@ import { CalciteButton, CalciteIcon, CalciteSlider } from "@esri/calcite-compone
 
 ### Dist build
 
+<!-- TODO: Get suggestions for what this should be replaced with -->
```

Comment:
I think we can link to https://developers.arcgis.com/calcite-design-system/get-started/#distribution here. Applies to similar build sections.